code/app/src/main/java/com/example/sigma_blue/Tag.java

```diff
@@ -0,0 +1,55 @@
+package com.example.sigma_blue;
+
+import android.graphics.Color;
+
+/**
+ * Stores information about a single tag.
+ */
+public class Tag implements Comparable<Tag>{
+    private String tagText;
+    private Color colour;
+
```

Comment:
Should we have a colorless implementation where the Tag is constructed without a Color?

Reply:
I mean we could set it to white by default

Reply:
I think a better solution is to just use the constructor with an int and use new Tag("name", WHITE). I don't think it really justifies another constructor.

Reply:
> I think a better solution is to just use the constructor with an int and use new Tag("name", WHITE). I don't think it really justifies another constructor.

Might be something that the factory pattern could help with, but might not be necessary as well.